routes/v1/flowersStore/flowersStore_router.go

```diff
@@ -0,0 +1,36 @@
+package flowersStore_router
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	flowersStore_controllers "github.com/ice-cream-backend/controllers/v1/flowersStore"
+	users_controllers "github.com/ice-cream-backend/controllers/v1/users"
+	errors_models "github.com/ice-cream-backend/models/v1/errors"
+	flowersStore_models "github.com/ice-cream-backend/models/v1/flowersStore"
+	"github.com/ice-cream-backend/utils"
+)
+
+func BuyNewFlower(w http.ResponseWriter, r *http.Request) {
+	utils.EnableCors(&w)
+
+	var flowersStore flowersStore_models.FlowersStore
+	_ = json.NewDecoder(r.Body).Decode(&flowersStore)
+
+	_, err := flowersStore_controllers.BuyNewFlower(flowersStore)
+	if err != nil {
+		var iceCreamError errors_models.IceCreamErrors
+		iceCreamError.Error = err.Error()
+		iceCreamError.Info = "Invalid gardenId provided"
```

Comment:
Can you update the error reason? I'm guessing it'll be something like "Could not buy flower" or something like that.

Reply:
Thank you for your check! I changed string to "Invalid flowersStore request provided" 🙏